{"name": "Invalid value for param - roomSize in the Request Body and getting Incorrect output -- @1.6 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [0, 2], 'patches': 4}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628277072114, "stop": 1628277072115}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628277072115, "stop": 1628277072116}, {"name": "When Set invalid request Body { \"roomSize\" : [5, 5], \"roomSize\" : [4, 4], \"coords\" : [1, 2], \"patches\" : [[2, 2]], \"instructions\" : \"NNESEESWNWW\" }", "status": "passed", "start": 1628277072116, "stop": 1628277072116}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628277072117, "stop": 1628277072130}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [0, 2], 'patches': 4}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628277072130, "stop": 1628277072131}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5, 5], \"roomSize\" : [4, 4], \"coords\" : [1, 2], \"patches\" : [[2, 2]], \"instructions\" : \"NNESEESWNWW\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628277072112, "stop": 1628277072132, "uuid": "a6fc5767-8593-449c-984e-dc619aa44675", "historyId": "e9ad4984d29bcadba0154e2330247112", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for roomSize params: Invalid value for param - roomSize in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for roomSize params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}